### PR TITLE
Manage Attendees

### LINKED ISSUE #101: Manage attendee list on a dedicated view
Create a new view for hosts that shows the attendees in a table view with action buttons (e.g. Make host, remove) and sortable headers.

### PR DESCRIPTION
**Pull request by trymebytes**
Fixes #101 
**WIP**

**Screenshot**
image

image

### CODE REVIEW

File: templates/events-attendees.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Events list page.
+ */
+
+namespace Wporg\TranslationEvents;
+
```
Comment (psrpinto) [inline templates/events-attendees.php:7]:
```suggestion
use Wporg\TranslationEvents\Event\Event;

/** @var Event $event */
/** @var bool $is_active_filter */
```

File: templates/events-attendees.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+gp_tmpl_load( 'events-header', get_defined_vars(), __DIR__ );
+
+
+
+?>
+<div class="event-page-wrapper">
+	<div class="event-details-stats">
+	<a href="<?php echo esc_url( Urls::event_details( $event->id() ) ); ?>" class="view-event-page">Go to event page</a>
+	<ul class="event-attendees-filter">
+		<li><a class="<?php echo ( ! $is_active_filter ) ? 'active-filter' : ''; ?>" href="<?php echo esc_url( Urls::event_attendees( $event_slug ) ); ?>"><?php esc_html_e( 'All attendees', 'gp-translation-events' ); ?></a></a></li>
```
Comment (psrpinto) [inline templates/events-attendees.php]:
```suggestion
		<li><a class="<?php echo ( ! $is_active_filter ) ? 'active-filter' : ''; ?>" href="<?php echo esc_url( Urls::event_attendees( $event->slug() ) ); ?>"><?php esc_html_e( 'All attendees', 'gp-translation-events' ); ?></a></a></li>
```

**Review by psrpinto — APPROVED** [review]
Works great, well done!

**Comment by trymebytes** [comment]
Thank you for catching these @psrpinto !

**Comment by psrpinto** [comment]
Just to be clear, `$event_slug` would also work because the variable is defined on the route, but I find that it's more convenient/readable to use the getters on `$event` as then there's no need to declare `$event_slug` (or other variables) on the template. Just a detail :)

**Review by psrpinto — APPROVED** [review]

**Pull request merged**